File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/ServerOrderIP.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// An IP block option available when ordering a dedicated server.
  /// </summary>
  [DataContract]
  public class ServerOrderIP {
    /// <summary>
    /// IP ID.
    /// </summary>
    /// <value>IP ID.</value>
    [DataMember(Name="id", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "id")]
    public string Id { get; set; }

    /// <summary>
    /// IP price.
    /// </summary>
    /// <value>IP price.</value>
    [DataMember(Name="price", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "price")]
    public int? Price { get; set; }

    /// <summary>
    /// IP image.
    /// </summary>
    /// <value>IP image.</value>
    [DataMember(Name="img", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "img")]
    public string Img { get; set; }

    /// <summary>
    /// Short description of the IP.
    /// </summary>
    /// <value>Short description of the IP.</value>
    [DataMember(Name="short_desc", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "short_desc")]
    public string ShortDesc { get; set; }

    /// <summary>
    /// Long description of the IP.
    /// </summary>
    /// <value>Long description of the IP.</value>
    [DataMember(Name="long_desc", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "long_desc")]
    public string LongDesc { get; set; }

    /// <summary>
    /// Quantity of IPs.
    /// </summary>
    /// <value>Quantity of IPs.</value>
    [DataMember(Name="qty", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "qty")]
    public string Qty { get; set; }

    /// <summary>
    /// Monthly price.
    /// </summary>
    /// <value>Monthly price.</value>
    [DataMember(Name="monthly_price", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "monthly_price")]
    public int? MonthlyPrice { get; set; }

    /// <summary>
    /// Display of IP price.
    /// </summary>
    /// <value>Display of IP price.</value>
    [DataMember(Name="price_display", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "price_display")]
    public string PriceDisplay { get; set; }

    /// <summary>
    /// Display of monthly IP price.
    /// </summary>
    /// <value>Display of monthly IP price.</value>
    [DataMember(Name="monthly_price_display", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "monthly_price_display")]
    public string MonthlyPriceDisplay { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ServerOrderIP {\n");
      sb.Append("  Id: ").Append(Id).Append("\n");
      sb.Append("  Price: ").Append(Price).Append("\n");
      sb.Append("  Img: ").Append(Img).Append("\n");
      sb.Append("  ShortDesc: ").Append(ShortDesc).Append("\n");
      sb.Append("  LongDesc: ").Append(LongDesc).Append("\n");
      sb.Append("  Qty: ").Append(Qty).Append("\n");
      sb.Append("  MonthlyPrice: ").Append(MonthlyPrice).Append("\n");
      sb.Append("  PriceDisplay: ").Append(PriceDisplay).Append("\n");
      sb.Append("  MonthlyPriceDisplay: ").Append(MonthlyPriceDisplay).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/server_order_ip.dart
part of swagger.api;

class ServerOrderIP {
  /* IP ID. */
  String id = null;
/* IP price. */
  int price = null;
/* IP image. */
  String img = null;
/* Short description of the IP. */
  String shortDesc = null;
/* Long description of the IP. */
  String longDesc = null;
/* Quantity of IPs. */
  String qty = null;
/* Monthly price. */
  int monthlyPrice = null;
/* Display of IP price. */
  String priceDisplay = null;
/* Display of monthly IP price. */
  String monthlyPriceDisplay = null;

  ServerOrderIP();

  @override
  String toString() {
    return 'ServerOrderIP[id=$id, price=$price, img=$img, shortDesc=$shortDesc, longDesc=$longDesc, qty=$qty, monthlyPrice=$monthlyPrice, priceDisplay=$priceDisplay, monthlyPriceDisplay=$monthlyPriceDisplay, ]';
  }

  ServerOrderIP.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    id = json['id'];
    price = json['price'];
    img = json['img'];
    shortDesc = json['short_desc'];
    longDesc = json['long_desc'];
    qty = json['qty'];
    monthlyPrice = json['monthly_price'];
    priceDisplay = json['price_display'];
    monthlyPriceDisplay = json['monthly_price_display'];
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'price': price,
      'img': img,
      'short_desc': shortDesc,
      'long_desc': longDesc,
      'qty': qty,
      'monthly_price': monthlyPrice,
      'price_display': priceDisplay,
      'monthly_price_display': monthlyPriceDisplay
     };
  }

  static List<ServerOrderIP> listFromJson(List<dynamic> json) {
    return json == null ? new List<ServerOrderIP>() : json.map((value) => new ServerOrderIP.fromJson(value)).toList();
  }

  static Map<String, ServerOrderIP> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ServerOrderIP>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ServerOrderIP.fromJson(value));
    }
    return map;
  }
}
